src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/LabelComponentSpec.scala

```diff
@@ -5,10 +5,11 @@ import java.time.Instant
 import java.util.UUID
 
 import org.broadinstitute.dsde.workbench.leonardo.model.{Cluster, ClusterStatus}
+import org.scalatest.FlatSpecLike
 
 import scala.util.Random
 
-class LabelComponentSpec extends TestComponent {
+class LabelComponentSpec extends TestComponent with FlatSpecLike {
```

Comment:
revert

Reply:
? this is right, i think. the spec type got removed from TestComponent

Reply:
Yeah, `TestComponent` doesn't assume a spec type anymore.

Reply:
Ah!  ok